lib/services/debug-service-base.ts

```diff
@@ -34,12 +34,16 @@ export abstract class DebugServiceBase extends EventEmitter implements IDeviceDe
 
 	protected getChromeDebugUrl(debugOptions: IDebugOptions, port: number): string {
 		// corresponds to 55.0.2883 Chrome version
+		// SHA is taken from https://chromium.googlesource.com/chromium/src/+/55.0.2883.100
+		// This SHA is old and does not support debugging with HMR.
+		// In case we want to stick with concrete SHA, get it from one of the tags https://chromium.googlesource.com/chromium/src/
+		// IMPORTANT: When you get the SHA, ensure you are using the `parent` commit, not the actual one.
+		// Using the actual commit will result in 404 error in the remote serve.
```

Comment:
Maybe `remote server` 😸

Reply:
Nope, `devtools/remote/serve_file` - that's why I used remote serve.